src/components/EditHomepage/EditHomepage.js

```diff
@@ -0,0 +1,273 @@
+              ? skills.map((skill, index) => (
+                  <Badge
+                    as={Button}
+                    onClick={() =>
+                      setSkills([...skills.filter((s) => s !== skill)])
+                    }
+                    pill
+                    variant="info"
+                    style={{ marginLeft: "5px", marginRight: "5px" }}
+                    key={index}
+                    className="skillsBadges"
+                  >
+                    {skill}
+                  </Badge>
+                ))
+              : null}
+          </Container>
+        </Form.Group>
+        <Form.Group controlId="formGroupProjectsandExpereince">
+          <Form.Label>Projects and Expereince</Form.Label>
+          <Form.Control
+            as="textarea"
+            rows="3"
+            placeholder="Tell everyone about past projects and professional successes"
+            value={experience}
+            onChange={(event) => setExperience(event.target.value)}
+          ></Form.Control>
+        </Form.Group>
+        <Form.Group controlId="formGroupBio">
+          <Form.Label>More about you</Form.Label>
+          <Form.Control
+            as="textarea"
+            rows="3"
+            placeholder="More about you! What makes you a great teammate?"
+            value={bio}
+            onChange={(event) => setBio(event.target.value)}
+          ></Form.Control>
+        </Form.Group>
+        <Form.Group controlId="formGroupWebsites" className="mt-5">
+          <Form.Label>What are some websites that show your skills?</Form.Label>
+          <Form.Control
+            type="text"
+            placeholder="Press enter to add the url your page"
+            value={website}
+            onChange={(event) => setWebsite(event.target.value)}
+          ></Form.Control>
+        </Form.Group>
+        <Button
+          type="submit"
+          onClick={submitWebsite}
+          style={{
+            margin: "10px",
+            backgroundColor: "#6610f2",
+            border: "#6610f2",
+          }}
+        >
+          Enter Website
+        </Button>
+        <Container as={Row}>
+          <ListGroup horizontal style={{ flexWrap: "wrap", margin: "15px" }}>
+            {websites.length > 0
+              ? websites.map((website, index) => {
+                  return (
+                    <ListGroup.Item
+                      className="websiteEntries"
+                      as={Button}
+                      onClick={() =>
+                        setWebsites([...websites.filter((w) => w !== website)])
+                      }
+                      variant="info"
+                      style={{ marginLeft: "5px", marginRight: "5px" }}
+                      key={index}
+                    >
+                      <a href={`http://${website}`}>{website}</a>
+                    </ListGroup.Item>
+                  );
+                })
+              : null}
+          </ListGroup>
+        </Container>
```

Comment:
This might be a good to separate it to a component since the form itself is big enough already